src/js/providers/html5.js

```diff
@@ -455,6 +456,41 @@ define([
             if (_source.preload) {
                 _videotag.setAttribute('preload', _source.preload);
             }
+
+            // if playlist item contains .vtt tracks, load them
+            if (utils.isIOS() && _itemTracks) {
+                _setupSideloadedTracks();
+            }
+        }
+
+        function _setupSideloadedTracks() {
+            // cleanup dom
+            if (_videotag.childNodes) {
```

Comment:
I don't think you need this if statement. `while (_videotag.firstChild)` does it for you.